### DISCUSSION TITLE
Expressing a collection visually.

### DISCUSSION THREAD

**Discussion by KULTI1995** [firecmsco/firecms#426] [Q&A]
I have one fundamental question. Are you going to release it in the form of a MIT license, or are you doing it in a private repository for money.

![Screenshot_2022-11-06-16-24-30-074_com android chrome](https://user-images.githubusercontent.com/14282128/200179484-0ad7cb68-f6c4-43ce-972d-ede0d8d38a6a.jpg)

**Comment by fgatti675** [comment]
This was asked by yourself and answered here already https://github.com/Camberi/firecms/discussions/422#discussioncomment-4060922
